src/notifications/components/InboxButton.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+import styles from './InboxButton.css'
+
+const showInboxClass = showInbox =>
+    classNames({
+        [styles.inbox]: true,
```

Comment:
Not really important, but more of an FYI when you use the `classnames` package: you can just write `classNames(styles.inbox, { [styles.activeInbox]: showInbox })` for classes that are always there